src/wp-includes/theme.php

```diff
@@ -147,6 +147,16 @@ function wp_clean_themes_cache( $clear_update_cache = true ) {
 	}
 }
 
+/**
+ * Clears the cache held by WP_Theme_JSON and WP_Theme_JSON_Resolver classes.
```

Comment:
It might be worth adding more documentation if this is a public function. I'm thinking along the lines of a brief detailing of the side effects calling this function will have.

Reply:
Thanks, I've added an extra line here to indicate what the function does. I found it hard to capture in a single sentence without falling into the trap of describing the particular use case with `registered_block_type`. Happy to expand on that if it's suitable, though.